src/components/PeopleTable.tsx

```diff
@@ -1,52 +1,166 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../types';
+import { PersonLink } from './personLink';
+import { SearchLink } from './SearchLink';
+import { SearchParams } from '../utils/searchHelper';
+import { SortTypes } from '../types/sortParams';
+
+type Props = {
+  setSearchWith:(params: SearchParams) => void,
+  people: Person[],
+  allPeople: Person[],
+  order: string,
+  sort: string,
+  searchParams: URLSearchParams,
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  setSearchWith,
+  people,
+  allPeople,
+  order,
+  sort,
+  searchParams,
+}) => {
+  const { personSlug } = useParams();
+
+  const isPersonInList = (personName: string) => (
+    allPeople.find(person => person.name === personName)
+  );
+
+  const getParent = (
+    personParent: string,
+  ) => {
+    const parent = isPersonInList(personParent);
+
+    return parent
+      ? (
+        <PersonLink
+          person={parent}
+          searchParams={searchParams}
+          selectedPerson={personSlug}
+        />
+      )
+      : personParent;
+  };
+
+  const handleSortOrder = (
+    event: React.MouseEvent<HTMLAnchorElement,
+    MouseEvent>, sortOrder: string,
+  ) => {
+    if (order && sort === sortOrder) {
+      setSearchWith({ sort: null, order: null });
+      event.preventDefault();
+    }
+  };
+
   return (
     <table
       data-cy="peopleTable"
       className="table is-striped is-hoverable is-narrow is-fullwidth"
     >
+
       <thead>
         <tr>
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
+              <SearchLink
+                onClick={(event) => handleSortOrder(event, SortTypes.Name)}
```

Comment:
you don't need it
it should work as a link - just pass the correct `params` prop, the rest logic already implemented inside SearchLink component